### PR TITLE
fix: Improve mobile view

### PR DESCRIPTION
**Pull request by anshg1214**
This PR fixes the case where BrainzPlayer hide the login button on homepage on the mobile

### CODE REVIEW

**Review by MonkeyDo — COMMENTED** [review]
I suppose in this case it would make more sense to hide BrainzPlayer on the homepage, no?
There's no music to play and I think it doesn't improve the homepage.

File: frontend/css/homepage.less [hunk 1]
```diff
@@ -459,7 +459,7 @@
         top: 0;
       }
       .homepage-info {
-        bottom: 1.5em;
+        bottom: 4.5em;
```
Comment (MonkeyDo) [inline frontend/css/homepage.less:462]: I think this should probably be reverted, on mobile phone in horizontal mode the bottom text comes butting in with the top text:
image

With the previous `1.5em` value:
image


Only applies to this `max-height` media query, the other one is fine

**Review by MonkeyDo — COMMENTED** [review]
Sweet!
One last little detail and we're good to merge :rocket:

File: frontend/css/homepage.less (hunk 1 above)
Comment (anshg1214) [reply frontend/css/homepage.less:462]: But, in the cases where we show the BrainzPlayer on the homepage, it fits perfectly
image

Comment (MonkeyDo) [reply frontend/css/homepage.less:462]: Ah yes, I forgot while testing that you can be playing music and then go to the landing page.

**Review by MonkeyDo — APPROVED** [review]
:rocket:

**Pull request merged**